copier.yml

```diff
@@ -5,6 +5,7 @@ _min_copier_version: "9"
 
 _tasks:
   - rm -f {% if ci != "Travis" %}.travis.yml{% endif %} .t2d.yml
+  - rm -f .prettierrc.yml .eslintrc.yml
```

Comment:
@PabloEForgeFlow these should be removed only for odoo_version >= 18, right? Can you adjust that?

Reply:
Done in 6f0c78888e983606697d4d4c8cf52bf4c3eec471

Reply:
I though they would be recreated, but yeah it's probably more clear this way.